docs/modules/examples.rst

```diff
@@ -0,0 +1,14 @@
+Example Usages
+==============
+
+Here are some ways to use our package in **your own bot!**
+
+A basic implementation
+----------------------
+
+.. code-block:: python
+    :linenos:
+
+    """
+    Insert example code here
```

Comment:
This needs content.

Reply:
No offense but not my area to resolve. Up to Logan or y'all to fill them how you see fit

Reply:
I don't disagree. CC @lgaan